mne/viz/ica.py

```diff
@@ -824,14 +824,14 @@ def _plot_ica_sources_evoked(evoked, picks, exclude, title, show, ica, labels=No
                 style = cat_styles[label_name]
                 label_props[label_idx] = (color, style)
 
-    for exc_label, ii in zip(exclude_labels, picks):
-        color, style = label_props[ii]
+    for pick_idx, (exc_label, i_pick) in enumerate(zip(exclude_labels, picks)):
```

Comment:
I think `pick_idx` and `i_pick` are too similar / confusing.  If I'm reading the code correctly, I think changing `i_pick` to just `pick` is the right repair. WDYT?

Reply:
That sounds much better. Thanks for the suggestion, which I committed.